charts/motel-regional/values.yaml

```diff
@@ -5,6 +5,26 @@ global:
 cert-manager:
   enabled: true
   email: mail@example.net
+external-dns:
+  enabled: false
+  provider:
+    name: aws
+  aws_static_credentials:
+    secretName: external-dns-aws-credentials
+    secrets:
+      aws_access_key_id: <access_key_id>
```

Comment:
Yes, this is an optional section, it can removed. In this case the secret has to be created externally.
I'll remove it from example and add to the readme.

Reply:
@Kshatrix removed